src/io/GeantParticle.hh

```diff
@@ -0,0 +1,56 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file GeantParticle.hh
+//---------------------------------------------------------------------------//
+/*!
+ * \brief Stores Geant4 particle definition information
+ *
+ * Class used to store G4ParticleDefinition data from Geant4.
+ * GeantParticle objects are stored into a ROOT file by the app/geant-exporter
+ * and retrieved from said ROOT file by the GeantImporter class.
+ */
+//---------------------------------------------------------------------------//
+
+#pragma once
+
+#include <string>
+
+#include "base/Types.hh"
+
+namespace celeritas
+{
+struct GeantParticle
+{
+    std::string name;
+    ssize_type  pdg;
+    real_type   mass;
+    real_type   charge;
+    real_type   spin;
+    real_type   lifetime;
+    bool        is_stable;
+
+    GeantParticle();
+    GeantParticle(std::string name,
+                  ssize_type  pdg,
+                  real_type   mass,
+                  real_type   charge,
+                  real_type   spin,
+                  real_type   lifetime,
+                  bool        is_stable);
+
+    ~GeantParticle() = default;
+
+    GeantParticle operator()(std::string name,
+                             ssize_type  pdg,
+                             real_type   mass,
+                             real_type   charge,
+                             real_type   spin,
+                             real_type   lifetime,
+                             bool        is_stable);
```

Comment:
All of these methods should be unneeded -- you can default construct the GeantParticle's data members more clearly by having e.g.
```
real_type   mass = 0;
```
and construct  inline if needed with
```
GeantParticle foo{"blah", 123, 3, 4, 5, 6, 7, false};
```